frontend/institution/base_institution_page.html

```diff
@@ -62,28 +62,28 @@
                                         <md-divider style="margin-top: 10px"></md-divider>
                                     </div>
                                     <md-menu-item>
-                                        <md-button ng-class="institutionCtrl.inStateView('timeline')?'option-selected-inst':''"
+                                        <md-button ng-class="institutionCtrl.inStateView('timeline')?'option-selected-left-bar':''"
```

Comment:
extract this logic to the controller, as you did previously on the institution page.